pandas/_libs/hashtable.pyx

```diff
@@ -44,7 +44,7 @@ cdef int64_t NPY_NAT = util.get_nat()
 _SIZE_HINT_LIMIT = (1 << 20) + 7
 
 
-cdef size_t _INIT_VEC_CAP = 128
+cdef ssize_t _INIT_VEC_CAP = 128
```

Comment:
ssize_t all the way down, to match above.

Reply:
I think Py_ssize_t here as well

Reply:
done.